.changeset/rotten-candles-train.md

```diff
@@ -0,0 +1,5 @@
+---
+"@rocket.chat/meteor": patch
+---
+
+Fixed toggle switch that enables/disables a Business Hour to reflect its current value.
```

Comment:
> Verb: Start with a verb in the third-person singular present tense.
https://developer.rocket.chat/docs/development-workflow#adding-changeset-to-your-pull-request
```suggestion
Fixes the "Enabled" toggle always being displayed as active when editing a business hour.
```